.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
 
 script:
   - pipenv run pytest tests
+  - pipenv run pytest --doctest-modules 
```

Comment:
I think you can acheive this by adding a configuration file in the project's root. 

this way the doctest will always be run, even in local